lib/scanner/walk.go

```diff
@@ -364,7 +364,7 @@ func (w *walker) walkRegular(ctx context.Context, relPath string, info fs.FileIn
 		if curFile.IsEquivalent(f, w.IgnorePerms, true) {
 			return nil
 		}
-		if curFile.Invalid {
+		if curFile.IsIgnored() {
```

Comment:
I think of them as a sort of conflict because from the user viewpoint, on linux there exists a symlink at path X, while on windows there is nothing at path X. So if he creates a file on windows at X, this file will silently override the symlink on linux, which is non-apparent, as it was never actively removed by the user.

Reply:
You've convinced me. I wanted to make an exception for `MustRescan`, but the circumstances under which we set that flag (something changed unexpectedly while pulling) means it would end up a conflict regardless so the distinction doesn't matter.